app/services/api/webflow-helpers.ts

```diff
@@ -173,12 +187,16 @@ const convertScheduleToCardProps = (schedule: ScheduledEvent): ScheduleCardProps
       const workshop = schedule.workshop
       return {
         variant: "workshop",
-        formattedStartTime: formatDate(schedule["day-time"], "h:mm aaa"),
+        formattedStartTime: formatDate(
+          schedule["day-time"],
+          schedule["end-time"] ? "h:mm" : "h:mm aaa",
```

Comment:
Are `Recurring` types the only ones we care about augmenting the date format? It looks like we're making the assumption that they will never be the same.

Reply:
regarding the dates, we're deciding based on the UI. not the best but it gets the UI in place as needed